test/extensions/implementation/backpack.py

```diff
@@ -190,3 +193,35 @@ def diag_h_batch(self):
             diag_h_batch = [p.diag_h_batch for p in self.problem.model.parameters()]
 
         return diag_h_batch
+
+    def ggn(self):
```

Comment:
Public method without docstring. So either 
- add docstring
- or `# noqa: D102` and docstring to parent class

Same applies to the other methods

Reply:
Add missing docs for public functions that are not in `base.py`. Add `noqa` for functions that are already documented there. Adding the file to `fully_documented.txt`.